cli/types/tslint.json

```diff
@@ -38,7 +38,10 @@
       "./mocha/*",
       "./minimatch/*",
       "./sinon/**/*",
-      "./sinon-chai/*"
+      "./sinon-chai/*",
+      // Copied from net-stubbing and renamed to a declaration file. Since it's not originally
+      // a declaration file, we need to exclude it from linting.
+      "./net-stubbing.d.ts"
```

Comment:
Fixed #22417 as a part of this work. Angular crawls all of the imported types and emits warnings due to this file not being included in the compilation. `.d.ts` files are fine and it's what this file should be anyway. I had to ignore this file in the `tslint.json` as `dtslint` complains. Since this file is originally a normal ts file, fixing the dtslint warnings causes ts warnings, so ignoring seemed best.